06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/README.md

```diff
@@ -0,0 +1,13 @@
+#### Expected Output
+- **Using**: HTML, CSS & JS
+- **Deliver**: a folder containing the template for the proposed frontend of your project
+- **With**: *independently*
+
+#### Output
+
+- [Link to folder containing template for proposed frontend of project](https://github.com/andela-eshaibu/inverted-index/tree/feature/139952859/frontend-template/UI)
+
+#### Screenshots
+
+**_Screenshot of proposed frontend template_**
+ ![frontend](https://cloud.githubusercontent.com/assets/25608317/23165226/5735e49a-f83b-11e6-8b6e-084e903b2247.JPG)
```

Comment:
Great Job!

You might want to rethink the idea of having multiple `Create Index` button elements on the left. What happened when 50 files are uploaded?

I'll accept this but expect that you fix this.